src/components/element-tile.tsx

```diff
@@ -26,7 +26,7 @@ const elementTileStyle = css`
 
 export const ElementTile = ({ children, componentObj }: any) => {
 	return (<div
-		className={elementTileStyle}
+		className={componentObj.className === undefined ? elementTileStyle: componentObj.className }
```

Comment:
What does this have to do with icons?

Reply:
a class name from a componentObj??

Shouldn't className then be set by passing it to ElementTile in place where you create the list? And we could use cx to add styles to existing elementTileStyle to override.